### ISSUE TITLE
The application in the dashboard does not get created when selecting certain options from CLI

### ISSUE THREAD

**Issue by geekchick** [Vonage/vonage-cli#24]
Hi! The application in the dashboard does not get created in the following scenarios from the command line. Please note the only differences are the **Application Name** and **Allow use of data for AI training?**:

% vonage apps:create
✔ Application Name … atomic_perch
✔ Select App Capabilities › Voice, Messages, RTC, VBC
✔ Create voice webhooks? … yes
✔ Answer Webhook - URL … https://example.com/webhook_name
✔ Answer Webhook - Method › POST
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Create messages webhooks? … yes
✔ Inbound Message Webhook - URL … https://example.com/webhook_name
✔ Inbound Message Webhook - Method › POST
✔ Status Webhook - URL … https://example.com/webhook_name
✔ Status Webhook - Method › POST
✔ Create RTC webhook? … yes
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Allow use of data for AI training? Read data collection disclosure - https://help.nexmo.com/hc/en-us/articles/4401914566036 … no

Creating Application... done

**AND...**

 % vonage apps:create
✔ Application Name … negative_giraffe
✔ Select App Capabilities › Voice, Messages, RTC, VBC
✔ Create voice webhooks? … yes
✔ Answer Webhook - URL … https://example.com/webhook_name
✔ Answer Webhook - Method › POST
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Create messages webhooks? … yes
✔ Inbound Message Webhook - URL … https://example.com/webhook_name
✔ Inbound Message Webhook - Method › POST
✔ Status Webhook - URL … https://example.com/webhook_name
✔ Status Webhook - Method › POST
✔ Create RTC webhook? … yes
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Allow use of data for AI training? Read data collection disclosure - https://help.nexmo.com/hc/en-us/articles/4401914566036 … yes

Creating Application... done

**Comment by kellyjandrews** [comment]
Ok I finally uncovered the error - and should be able to get this sorted fairly quickly. 

```shell
❯ bin/run apps:create   
✔ Application Name … dying_halibut
✔ Select App Capabilities › Voice, Messages, RTC, VBC
✔ Create voice webhooks? … yes
✔ Answer Webhook - URL … https://example.com/webhook_name
✔ Answer Webhook - Method › POST
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Create messages webhooks? … no
✔ Create RTC webhook? … yes
✔ Event Webhook - URL … https://example.com/webhook_name
✔ Event Webhook - Method › POST
✔ Allow use of data for AI training? Read data collection disclosure - https://help.nexmo.com/hc/en-us/articles/4401914566036 … yes
Creating Application... !
 ›   Error: The request failed due to validation errors
```

**Comment by kellyjandrews** [comment]
Verified and fixed in @vonage/cli/1.0.0-beta.15 linux-x64 node-v12.18.2
